src/Microsoft.Azure.SignalR.Management/HubContext/StreamingManagerAdapter.cs

```diff
@@ -77,7 +70,7 @@ public override async Task SendStreamAsync<TItem>(string connectionId, string st
 
     private async Task SendErrorAsync(string connectionId, string streamId, Exception ex, CancellationToken cancellationToken)
     {
-        var message = _negotiationOptions.EnableDetailedErrors ? ex.Message : "An error occurred.";
-        await _lifetimeManager.SendStreamCompletionAsync(connectionId, streamId, message, cancellationToken);
+        _logger.LogError(ex, "An error occurred while sending stream items.");
```

Comment:
is it an intentional error log?

Reply:
it is server-side error.
For example:
```cs
_ = SendStreamAsync("connectionId", "streamId", GetStream());

public async IAsyncEnumerable<int> GetStream()
{
   await ...
   throw new InvalidOperationException("Ops!");
}
```
we will log the "Ops".